Code/Tools/ProjectManager/Source/ProjectManagerDefs.h

```diff
@@ -30,6 +30,7 @@ namespace O3DE::ProjectManager
     static const QString ProjectCMakeCommand = "cmake";
     static const QString ProjectCMakeBuildTargetEditor = "Editor";
 
-    static const QString RepoTimeFormat = "dd/MM/yyyy hh:mmap";
+    static const QString RepoTimeFormat = "MM/dd/yyyy hh:mmap";
+    static const QString CanonicalRepoUri = "https://canonical.o3de.org";
```

Comment:
A better format that doesn't have to worry about localization is the [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) format. It avoids the issues with day-major(rest of the world) vs month-major(US) dates, by specify the date as `yyyy-mm-dd` so it goes from most general date value(year) to the most specific(day)